### PR TITLE
Update radar height and zoom

### PR DESCRIPTION
**Pull request by greg-does-weather**
## What does this PR do? 🛠️

- Updates the radar zoom level to start at 8 (one zoomed out)
- Updates the radar to be a 1:1 aspect ratio at desktop sizes and 3:4 at tablet and below

## Screenshots (if appropriate): 📸

Desktop:
Screenshot 2024-04-11 at 11 40 21 AM

Mobile:
Screenshot 2024-04-11 at 11 40 35 AM

### CODE REVIEW

File: web/themes/new_weather_theme/assets/sass/components/_radar.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+@use "uswds-core" as *;
+
+.wx-radar-container {
+  aspect-ratio: 1/1;
+}
+
+@include at-media-max("tablet") {
```
Comment (igorkorenfeld) [inline web/themes/new_weather_theme/assets/sass/components/_radar.scss:7]: `at-media-max` is cool to know about! 

**non-blocking:** I am totally fine with leaving it this way, but the thought occurred to me that for other components we style mobile-first, and do the media query for the larger sizes.

**Review by igorkorenfeld — APPROVED** [review]
Sizes look good to me! One minor non-blocking thought.

File: web/themes/new_weather_theme/assets/sass/components/_radar.scss (hunk 1 above)
Comment (greg-does-weather) [reply web/themes/new_weather_theme/assets/sass/components/_radar.scss:7]: I'd need to check if there's an equivalent `at-media-min` in order to do that, but good call! I'll take a look.

Comment (greg-does-weather) [reply web/themes/new_weather_theme/assets/sass/components/_radar.scss:7]: Oh, `at-media` is the min version. Ha. Update incoming.

Comment (greg-does-weather) [reply web/themes/new_weather_theme/assets/sass/components/_radar.scss]: @igorkorenfeld Would you ask @partly-igor to take a look when he's got time? :trollface:

**Review by partly-igor — APPROVED** [review]
That other person made really good points and I agree with all of them.

**Pull request merged**